.shippable.yml

```diff
@@ -77,7 +77,8 @@ build:
            echo "Dumping the capability database in case we are affected by #9992"
            cat $HOME/.cache/zephyr/ToolchainCapabilityDatabase.cmake
          fi;
-      - rm -rf ccache $HOME/.cache/zephyr
+      - rm -rf ccache
+      - rm $HOME/.cache/zephyr/*
```

Comment:
at this stage, cmake will not be writing anything to this directory the build process is already over. Not sure how this is going to help.

Reply:
I am not familiar with the CI environment, but I was working under the assumption that two different jobs will re-use the same $HOME/.cache/ directory.

Reply:
@nashif : Is it possible for one job to be clearing the cache while another is populating it? Or does $HOME/.cache exist uniquely per-job?

Reply:
jobs are run on different instances, they do not share the same build directories.